src/d1to2fix/visitor.d

```diff
@@ -169,10 +169,63 @@ public final class TokenMappingVisitor : ASTVisitor
         }
     }
 
-    /*
-        helper methods
-     */
+    /**
+        If the node has a delegate, add the index at which to place
+        the `scope` to `this.token_mappings.scope_delegates`
+
+        Params:
+            type = parameter type
+            token = parameter name token
+    **/
```

Comment:
I have been using Phobos style for comments in d1to2fix originally but forgot about it when adding new stuff. Which way would you prefer to unify it?